share/templates/03-Chamber_Music/05-Brass_Quartet.mscx

```diff
@@ -27,6 +27,9 @@
     <metaTag name="workTitle"></metaTag>
     <Order id="brass-ensemble" customized="1">
       <name>Brass Ensemble</name>
+      <instrument id="flugelhorn">
+        <family id="flugelhorns">Flugelhorns</family>
+      </instrument>
```

Comment:
The <code>Instrument</code> is required for the actual instruments used in the template. Since you didn't extend the **template** with a flugelhorn but only extended the **order** these line are not needed and better left out.

The function of these lines is save which instruments actually belong to a family for **this** score. By saving these lines in the score, score will not change when somebody decides the change family definition in <code>instrument.xml</code>